test/parallel/test-fs-write-sync.js

```diff
@@ -1,23 +1,23 @@
 'use strict';
-var common = require('../common');
-var assert = require('assert');
-var path = require('path');
-var fs = require('fs');
-var fn = path.join(common.tmpDir, 'write.txt');
+const common = require('../common');
+const assert = require('assert');
+const path = require('path');
+const fs = require('fs');
+const fn = path.join(common.tmpDir, 'write.txt');
 
 common.refreshTmpDir();
 
-var foo = 'foo';
-var fd = fs.openSync(fn, 'w');
+const foo = 'foo';
+const fd = fs.openSync(fn, 'w');
 
-var written = fs.writeSync(fd, '');
+const written = fs.writeSync(fd, '');
 assert.strictEqual(0, written);
 
 fs.writeSync(fd, foo);
 
-var bar = 'bár';
-written = fs.writeSync(fd, Buffer.from(bar), 0, Buffer.byteLength(bar));
-assert.ok(written > 3);
+const bar = 'bár';
+const written2 = fs.writeSync(fd, Buffer.from(bar), 0, Buffer.byteLength(bar));
+assert.ok(written2 > 3);
```

Comment:
Changed it to use `let` in 8dc500710, although without the extra block. I hope that's OK.